ghc-mod.cabal

```diff
@@ -94,8 +94,12 @@ Extra-Source-Files:     ChangeLog
                         test/data/stack-project/app/*.hs
                         test/data/stack-project/src/*.hs
                         test/data/stack-project/test/*.hs
-                        bench/data/simple-cabal/simple-cabal.cabal
-                        bench/data/simple-cabal/*.hs
+                        test/data/options-cradle/ghc-mod.options
+                        test/data/options-cradle/ghc-mod.package-db-stack
+                        test/data/options-cradle/package-db-a/package.cache
```

Comment:
Okay, so you *don't* want to check in package.cache into repo, since it's ghc-specific binary format which can (and will) change, and your tests will fail for no good reason with something like
`GHC.PackageDb.readPackageDb: inappropriate type (not a ghc-pkg db file, wrong file magic number)`